_cookies_from_headers breaks on Python 2 if python-future is installed

(I have also reported this at PythonCharmers/python-future#564)

If [python-future](https://github.com/PythonCharmers/python-future) is installed, the `test_cookies_from_headers` test fails on Python 2 with the following error:
```
test_responses.py:1119: in test_cookies_from_headers
    cookiejar = responses._cookies_from_headers(headers)
responses.py:122: in _cookies_from_headers
    resp_cookie.load(headers["set-cookie"])
/usr/lib/python2.7/Cookie.py:643: in load
    for k, v in rawdata.items():
E   AttributeError: 'unicode' object has no attribute 'items'
```
This happens because python-future provides a `http.cookies` module which just imports the Python 2 `Cookie` module which doesn't support Unicode strings (see PythonCharmers/python-future#564 for more information). The [`_cookies_from_headers` function](https://github.com/getsentry/responses/blob/6f682b89e373cb05e5535f098bfa38cd05282382/responses.py#L117) uses the Python 3 code path because a `http.cookies` module is available, but it fails because it doesn't support Unicode strings.

--- a/responses/__init__.py
+++ b/responses/__init__.py
@@ -133,13 +133,13 @@ def _ensure_str(s):
 
 def _cookies_from_headers(headers):
     try:
-        import http.cookies as cookies
+        import http.cookies as _cookies
 
-        resp_cookie = cookies.SimpleCookie()
+        resp_cookie = _cookies.SimpleCookie()
         resp_cookie.load(headers["set-cookie"])
 
         cookies_dict = {name: v.value for name, v in resp_cookie.items()}
-    except ImportError:
+    except (ImportError, AttributeError):
         from cookies import Cookies
 
         resp_cookies = Cookies.from_request(_ensure_str(headers["set-cookie"]))

